ui/apps/dashboard/tsconfig.json

```diff
@@ -26,7 +26,6 @@
     "./sentry.edge.config.ts",
     "./sentry.server.config.ts",
     "../../packages/components/src/Menu/MenuItem.tsx",
-    "../dev-server-ui/src/components/Layout/Logo.tsx",
```

Comment:
@jacobheric I saw that you added this in #1642.

I can't see where it's being used within the dashboard, but this was causing build errors there as I added usage of `dev-server-ui/src/store/devApi.ts` into that file.

Highlighting to check that this is okay to remove. If it _is_ being used across multiple apps that I can move the `useInfoQuery()` code to another location and keep it pure.

Reply:
Fine to remove. I didn't mean to commit that. Sorry about that.